etcd-manager/Makefile

```diff
@@ -41,12 +41,19 @@ goimports:
 
 .PHONY: build-etcd-manager-amd64 build-etcd-manager-arm64
 build-etcd-manager-amd64 build-etcd-manager-arm64: build-etcd-manager-%:
+	# go build -o etcd-manager cmd/etcd-manager
 	${BAZEL} build ${BAZEL_FLAGS} --platforms=@io_bazel_rules_go//go/toolchain:linux_$* //cmd/etcd-manager:etcd-manager
 
 .PHONY: push-etcd-manager
 push-etcd-manager:
-	${BAZEL} run ${BAZEL_FLAGS} --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64 //images:push-etcd-manager
-	${BAZEL} run ${BAZEL_FLAGS} --platforms=@io_bazel_rules_go//go/toolchain:linux_arm64 //images:push-etcd-manager
+	echo $(REGISTRY_NAME)
```

Comment:
Why is this change required?

Reply:
The echo is not required indeed and was only used for debugging a script I was making which purpose was to build and push to the registry all the images needed by kOps across several repositories.
If you're talking about why I got rid of the Bazel commands it's because they required me to write BUILD.bazel files for the sdk and it seems like a lot of work that I was not sure was going to pay at the time. But now that I know that it works, I will eventually need to write these Bazel files.

Reply:
@Mia-Cross The `The BUILD.bazel` files are auto-generated by running `make vendor`.